### PR TITLE
switch pixelLess iteration to use CKF instead of mkFit

### PR DESCRIPTION
**Pull request by slava77**
Discussion in PPD Aug 4
https://indico.cern.ch/event/1187803/#43-discussion-of-reversion-to
appears to be converging to the decision to switch the pixelLess iteration to use CKF instead of mkFit.

This PR makes the switch and removes Run3_ckfPixelLessStep, which is now identical to Run3
(I checked this using the run3 wf 11834.0 expanded config for step3).

@cms-sw/tracking-pog-l2

### CODE REVIEW

File: Configuration/Eras/python/ModifierChain_trackingMkFitProd_cff.py [hunk 1]
```diff
@@ -16,5 +15,4 @@
     trackingMkFitHighPtTripletStep,
     trackingMkFitDetachedQuadStep,
     trackingMkFitDetachedTripletStep,
-    trackingMkFitPixelLessStep,
```
Comment (mmusich) [inline Configuration/Eras/python/ModifierChain_trackingMkFitProd_cff.py:19]: Shall we comment it, hoping to put it back ~ soon?

Comment (slava77) [reply Configuration/Eras/python/ModifierChain_trackingMkFitProd_cff.py:19]: IIRC, comments are usually not advised by reco: if some change is expected in the code soon, the usual advice is to make an issue in github

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
@slava77 
I have not followed up closely. I assume this PR is ready to go, right? I just call the test again as the last one is a week ago.

**Comment by slava77** [comment]
> @slava77 I have not followed up closely. I assume this PR is ready to go, right? I just call the test again as the last one is a week ago.

I do not expect any functional modifications in this PR.

The only pending item was a formal go-ahead from PPD/Validation that this is good to go for the default.
But that part is perhaps on the ORP part of the sign off.

**Comment by srimanob** [comment]
+Upgrade

From the upgrade side, this PR follows the agreement on switching pixelless iteration to use CKF instead of mkFit.

**Comment by kskovpen** [comment]
+pdmv

**Comment by perrotta** [comment]
+1
- As agreed at the Aug 16 ORP meeting

**Pull request merged**